### PR TITLE
NATIVE-586: Run gradle clean before building

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -108,7 +108,7 @@ jobs:
                 condition: << parameters.build_for_delivery >>
                 steps:
                     - run:
-                        command: bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE}
+                        command: ./gradlew clean | bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE}
```
Comment (maxammann) [inline .circleci/config.yml:111]: I think this should be `&`.

Comment (maxammann) [reply .circleci/config.yml:111]: Else the second command runs even if the first one fails

File: .circleci/config.yml [hunk 2]
```diff
@@ -108,7 +108,7 @@ jobs:
                 condition: << parameters.build_for_delivery >>
                 steps:
                     - run:
-                        command: ./gradlew clean | bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE}
+                        command: ./gradlew clean & bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE}
```
Comment (maxammann) [inline .circleci/config.yml:111]: Probably should be `&&`

Comment (klinzo) [reply .circleci/config.yml:111]:
```suggestion
                        command: ./gradlew clean && bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE}
```

Comment (klinzo) [reply .circleci/config.yml]: Yep, forgot to update the config.

**Pull request merged**